src/main/resources/db/migration/general/V4_0__modify_notification_add_snooze.sql

```diff
@@ -0,0 +1,52 @@
+ALTER TABLE "SHIFT_NOTIFICATION"
```

Comment:
I know we changed the casing on the table on last PR, but having table names and columns capitalised makes SQL harder to read as there's less visual queues between key words and named stuff.

Not saying this needs to be changed, just sharing thoughts